### DISCUSSION TITLE
训练集中是否要添加特殊的样本（缺失、模糊）？

### DISCUSSION THREAD

**Discussion by BeCoolMaker** [PaddlePaddle/PaddleOCR#13616] [Q&A]
请问，如果想要重新训练模型，在数据集中添加一些文字有缺失的样本会对模型效果有提升吗。

在训练集添加这种样本，会不会让模型对文字的结构认知出现错误？

**Comment by SWHL** [comment]
如果应用场景中有缺失和模糊的，训练时候添加会有提升的。
有可能会将模型训练崩了，具体还要实验来看
